src/Jellyfin.Extensions/Json/Converters/JsonDelimitedArrayConverter.cs

```diff
@@ -75,7 +65,29 @@ protected JsonDelimitedArrayConverter()
         /// <inheritdoc />
         public override void Write(Utf8JsonWriter writer, T[]? value, JsonSerializerOptions options)
         {
-            throw new NotImplementedException();
+            if (value is not null)
+            {
+                writer.WriteStartArray();
+                if (value.Length > 0)
+                {
+                    var toWrite = value.Length - 1;
+                    foreach (var it in value)
+                    {
+                        writer.WriteStringValue(it!.ToString());
```

Comment:
Shouldn't you properly handle null values here?

Reply:
How do you think they should be handled? `"null"` or `null` or dropped?

Reply:
I think `null` would typically be dropped